qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,30 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
-    repeat_times = 500
+    repeat_times = 1000
```

Comment:
500 is requested for windows test

Reply:
This change is agreed with @huangyum and @lijinlijin , Thanks!